frontend/src/scenes/experiments/Metrics/PrimaryGoalTrends.tsx

```diff
@@ -31,70 +35,245 @@ export function PrimaryGoalTrends(): JSX.Element {
-                buttonCopy="Add graph series"
-                showSeriesIndicator={true}
-                entitiesLimit={1}
-                showNumericalPropsOnly={true}
-                {...commonActionFilterProps}
+                                        setTrendsMetric({
+                                            metricIdx,
+                                            series,
+                                        })
+                                    }}
+                                    typeKey="experiment-metric"
+                                    buttonCopy="Add graph series"
+                                    showSeriesIndicator={true}
+                                    entitiesLimit={1}
+                                    showNumericalPropsOnly={true}
+                                    {...commonActionFilterProps}
+                                />
+                                <div className="mt-4 space-y-4">
+                                    <TestAccountFilterSwitch
+                                        checked={hasFilters ? !!currentMetric.count_query?.filterTestAccounts : false}
+                                        onChange={(checked: boolean) => {
+                                            setTrendsMetric({
+                                                metricIdx,
+                                                filterTestAccounts: checked,
+                                            })
+                                        }}
+                                        fullWidth
+                                    />
+                                </div>
+                                {isExperimentRunning && (
+                                    <LemonBanner type="info" className="mt-3 mb-3">
+                                        Preview insights are generated based on {EXPERIMENT_DEFAULT_DURATION} days of
+                                        data. This can cause a mismatch between the preview and the actual results.
+                                    </LemonBanner>
+                                )}
+                                <div className="mt-4">
+                                    <Query
+                                        query={{
+                                            kind: NodeKind.InsightVizNode,
+                                            source: currentMetric.count_query,
+                                            showTable: false,
+                                            showLastComputation: true,
+                                            showLastComputationRefresh: false,
+                                        }}
+                                        readOnly
+                                    />
+                                </div>
+                            </>
+                        ),
+                    },
+                    {
+                        key: 'exposure',
+                        label: 'Exposure',
```

Comment:
Oops, I didn't realize we accidentally dropped the exposure metric UI for Trends. I'm surprised we haven't gotten any tickets about that.

Reply:
Uhm we didn't? https://us.posthog.com/project/2/experiments/73149

<img width="457" alt="image" src="https://github.com/user-attachments/assets/a26d0f8d-19a5-4f1f-b180-b7211118c617" />

Reply:
@jurajmajerik You're right. I was looking at a data warehouse trends experiment where we did remove the UI element.